lark/lark.pyi

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+from typing import List, Dict, IO, Callable, Union, Optional, Literal
+from .visitors import Transformer
+from .lexer import Lexer, Token
+from .tree import Tree
+
+_Start = Union[None, str, List[str]]
+
+
+class Lark:
+
+    def __init__(
+        self,
+        grammar: Union[str, IO[str]],
+        *,
+        start: _Start = ...,
+        parser: Literal["earley", "lalr", "cyk"] = ...,
+        lexer: Optional[Lexer] = ...,
+        transformer: Optional[Transformer] = ...,
+        postlex: Optional[Literal["standard", "contextual"]] = ...,
```

Comment:
`postlex` is a callable object (a lexer proxy)

However
```python
  lexer: Optional[Literal["standard", "contextual", "dynamic", "dynamic_complete"]]
```

Reply:
What is `postlex` looks like?
I found an `Indenter` instance was used in there.
It's not exposed yet in my work.

Reply:
postlex is a Protocol with the method `process(self, stream: Iterator[Token]) -> Iterator[Token])`